src/applications/vaos/actions/newAppointment.js

```diff
@@ -314,7 +314,11 @@ export function openFacilityPage(page, uiSchema, schema) {
       }
 
       if (parentId && !eligibleFacilities?.length) {
-        recordEligibilityFailure('supported-facilities');
+        recordEligibilityFailure(
+          'supported-facilities',
+          typeOfCareId,
```

Comment:
So, I was looking at some other events code and it looks like we set `health-TypeOfCare` in other events to the type of care name, rather than the id: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/actions/newAppointment.js#L685. Probably makes sense to do the same here (and below).

Reply:
Done.